vim/load.ts

```diff
@@ -0,0 +1,63 @@
+import { Denops } from "../deps.ts";
+import { execute } from "./execute.ts";
+
+interface FileInfo {
+  isDirectory: boolean;
+}
+
+export async function load(
+  denops: Denops,
+  filePathInfo: URL,
+): Promise<void> {
+  const scriptPath = await setFile(denops, filePathInfo);
+
+  await execute(denops, `source ${scriptPath}`);
+}
+
+async function setFile(denops: Denops, filePathInfo: URL): Promise<string> {
+  const re = /.*\//;
+  const homeDir = await Deno.env.get("HOME") as string;
+
+  if (filePathInfo.protocol === "file:") {
+    const filename = filePathInfo.pathname.replace(re, "");
+    await makePluginDir(homeDir, denops.name);
+
+    await Deno.copyFile(
+      filePathInfo.pathname,
+      `${homeDir}/.cache/denops-std/${denops.name}/${denops.name}_${filename}`,
```

Comment:
*should*

Use `path` module to construct entire path string.

Reply:
**must**

We have to support Windows as well. I don't think `~/.cache` is proper directory to cache things in Windows.

Reply:
> We have to support Windows as well. I don't think `~/.cache` is proper directory to cache things in Windows.

I think `~/AppData/Local` is the way to go in Windows.